docs/quickstart/integrations/aws-lambda/quickstart-guide.mdx

```diff
@@ -348,3 +348,62 @@ handler = Mangum(app)
 
 </BackendTabs.TabItem>
 </BackendTabs>
+
+## Using with AWS Lambda Plugins / Extensions
+
+If you are using AWS Lambda plugins / extensions / anything that adds events to the lambda function (e:g `serverless-plugin-warmup`), then you may need to filter them and skip passing them to SuperTokens.
+
+These kind of events lack request details that SuperTokens expects, that are present in a normal request, and thus might lead to un-intended errors.
```

Comment:
remove the `that are present...` part. It's not necessary. just join the phrase on `and thus`.